x/feegrant/client/cli/query.go

```diff
@@ -133,14 +134,15 @@ $ %s query feegrant grants [grantee]
 // GetCmdQueryFeeGrantsByGranter returns cmd to query for all grants by a granter.
 func GetCmdQueryFeeGrantsByGranter() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "grants [granter]",
-		Args:  cobra.ExactArgs(1),
-		Short: "Query all grants by a granter",
+		Use:     "grants [granter]",
+		Aliases: []string{"grants-by-granter"},
```

Comment:
```suggestion
		Use:     "grants-by-granter [granter]",
```

This command was added recently, and is not released yet. There's no need for backwards-compatibility, and no need for aliases.